k8s/book-store-deployment.yaml

```diff
@@ -17,8 +17,8 @@ spec:
     spec:
       containers:
       - name: book-store
-        image: book-store
-        imagePullPolicy: IfNotPresent
+        image: IMAGE_VAR
```

Comment:
You can just construct the string right here, you have all the environment variables you need (I think, I read it in the docs).
`image: IMAGE_VAR` -> `image: gcr.io/$PROJECT_ID/book-store:$SHORT_SHA`.